package-lock.json

```diff
@@ -5290,9 +5290,9 @@
       "dev": true
     },
     "node_modules/brace-expansion": {
-      "version": "2.0.1",
-      "resolved": "https://registry.npmjs.org/brace-expansion/-/brace-expansion-2.0.1.tgz",
-      "integrity": "sha512-XnAIvQ8eM+kC6aULx6wuQiwVsnzsi9d3WxzV3FpWTGA19F621kwdbsAcFKXgKUHZWsy+mY6iL1sHTxWEFCytDA==",
+      "version": "2.0.2",
```

Comment:
This closes https://github.com/confluentinc/vscode/security/dependabot/26, the rest of the changes are for https://github.com/confluentinc/vscode/security/dependabot/27